File: Service/Models/RecalculateBracketResponseModel.cs
namespace TecmoTourney.Models;

public class RecalculateBracketResponseModel
{
    public List<int> CreatedGameResultIds { get; set; } = new();
    public List<int> SoftDeletedGameResultIds { get; set; } = new();
    public OddsGenerationStatusModel OddsGeneration { get; set; } = new();
    public bool Skipped { get; set; }
    public string? SkipReason { get; set; }
}

File: Client/tecmo-tourney/src/app/core/models/recalculate-bracket-response.model.ts
import { IOddsGenerationStatus } from './odds-generation-status.model';

export interface IRecalculateBracketResponse {
  createdGameResultIds: number[];
  softDeletedGameResultIds: number[];
  oddsGeneration: IOddsGenerationStatus;
  skipped: boolean;
  skipReason?: string | null;
}
